llvm_util/llvm2alive.cpp

```diff
@@ -64,28 +64,6 @@ string_view s(llvm::StringRef str) {
   } while (0)
 
 
-FastMathFlags parse_fmath(llvm::Instruction &i) {
```

Comment:
Moved to `utils.h/cpp` to make it visible to `known_fns.cpp`
I chose `utils.h/cpp` because utils seems to be the place where auxiliary functions are gathered